examples/dbc_io/main.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+#
+# Script testing dbc manipulation and io.
+#
+
+import os
+#import cantools
+import sys
+sys.path.append('/mnt/d/JQU/dbc/cantools')
+import cantools
+
+SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
+DBC_IO_DBC_PATH = os.path.join(SCRIPT_DIR, 'dbc_input.dbc')
+
+infile = DBC_IO_DBC_PATH
+outfile = infile.replace('_input', '_output')
+
+# Read the dbc
+db = cantools.db.load_file(infile)
+# Get a message to manipulate
+msg = db.get_message_by_name('TheMessage')
+# Manipulate a message attribute. Try/except will catch accesses to non existent attributes
+try:
+    print("Input cycle time: " + str(msg.dbc.attributes['GenMsgCycleTime'].value))

```

Comment:
I prefer: 

``` python
    print("Input cycle time:", msg.dbc.attributes['GenMsgCycleTime'].value)
```

Reply:
Do you mean just the space or do you want to get rid of the try/except to not fail silently? If the latter, would propose to output a meaningful message instead of pass. That way, the user gets an idea of the preferred usage (would say that is try/except).

Reply:
I just meant the print-statement.

Best case all exception thrown by cantools includes a descriptive error message. I'm fine with showing the backtrace to the user, but it's often nicer to catch the exception and just print its message (or another message if the exception message is unclear).

Reply:
I could catch and raise again.

Reply:
Ok, got it now. The concatenation. However, I also raise instead of pass now.